MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -59,6 +59,9 @@
 
 static NSString *const kDeviceIdClaimsValue = @"{\"access_token\":{\"deviceid\":{\"essential\":true}}}";
 
+static NSString *const kGlobalSignoutDarwinNotificationKey = @"FLWSignoutOccured";
```

Comment:
Should we define those in MSAL API somewhere?

Reply:
At this point, it seems that the Darwin notification solution is a targeted solution to only 1 partner app. If it seems that we will need to use it in other apps, I agree that it should be defined more generally in the MSAL API.

Reply:
After further discussions, we will need to expand this for other apps, so I created an MSID constant